ha/setup/ha_setup.py

```diff
@@ -344,6 +344,17 @@ def process(self):
             node_id = Conf.get(self._index, f"server_node.{machine_id}.node_id")
             self._confstore.set(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}", node_id)
 
+        node_id = Conf.get(self._index, f"server_node.{machine_id}.node_id")
+        cluster_id = Conf.get(self._index, f"server_node.{machine_id}.cluster_id")
+        site_id = Conf.get(self._index, f"server_node.{machine_id}.site_id")
+        rack_id = Conf.get(self._index, f"server_node.{machine_id}.rack_id")
+        storageset_id = Conf.get(self._index, f"server_node.{machine_id}.storageset_id")
+
+        # Update node map
+        node_map = {'cluster_id': cluster_id, 'site_id': site_id,
+                    'rack_id': rack_id, 'storageset_id': storageset_id}
+        self._confstore.set(f"/cortx/ha/system/cluster/node_map/{node_id}", str(node_map))
```

Comment:
I guess we can import and use the "const.COMPONENTS.NODE_MAP.value" here and replace the node_id in there.

Reply:
@indrajitzagade Yes we can, but for that we need to call prepare_key() from system_health.py to create Key. Is it really needed here ?
using "const.COMPONENTS.NODE_MAP.value" we get only node_map value not key.

Reply:
Yeah, lets call prepare_key and then use that key for storing the node map.

Reply:
Done.